src/Scope.ts

```diff
@@ -237,7 +237,7 @@ export class Scope {
         let namespaceLookup = {} as { [namespaceName: string]: NamespaceContainer };
         let files = this.getFiles();
         for (let file of files) {
-            for (let namespace of file.parser.namespaceStatements) {
+            for (let namespace of file.parser.references?.namespaceStatements ?? []) {
```

Comment:
Can't we assume `references` always exists?

Reply:
Yes, good point. Since the references object is created at construct time, and it's a getter which regenerates itself. I'll update these to assume it always exists.